docs/Using Fleet/enroll-chromebooks.md

```diff
@@ -24,6 +24,22 @@ By default, the hostname for a Chromebook host will be blank. The hostname can b
 ## Debugging ChromeOS
 To learn how to debug the Fleetd Chrome extension, visit [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Testing-and-local-development.md#fleetd-chrome-extension).
 
+## Potential issues and troubleshooting: 
```

Comment:
hmm rather than writing it as "here are problems." I think we should just assume that our customers are going to want to target installation. So this should just be presented as part of the normal process for doing the targeting. 

Sharon, we're meeting tomorrow to educate me a bit on this. I'm glad to take over this documentation writing to reflect the change above.